Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -25,21 +25,10 @@ private enum SearchListSection: Int{
     }
 }
 
-// A delegate to support clicking on rows in the view controller
-// This needs to be accessible to objc for UIViewControllers to call it
-@objc
-protocol SearchViewControllerDelegate: class {
-    func searchViewController(searchViewController: SearchViewController, didSubmitURL url: NSURL)
-}
-
-class SearchViewController: UIViewController {
-    weak var delegate: SearchViewControllerDelegate?
-    private var tableView = UITableView()
+class SearchViewController: HomePanelTableViewController {
```

Comment:
Yeah, I knew people would cry about this. All I care about is making the lists consistent so that we're not continually having to update them all. We can rename (I don't like that the Search stuff is in Frontend/Browser though, and I don't like that its owned by the BVC, which needs to move away from owning everything anyway).